login-system/classcreater.php

```diff
@@ -0,0 +1,22 @@
+<?php
+$_SESSION['classcode'] = $_POST['classcode'];
+$_SESSION['passcode'] = $_POST['passcode'];
```

Comment:
What is passcode?

Reply:
passcode is to allow only students of that class to join.